modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/vulcan/openapi/contributor/ObjectEntryOpenAPIContributor.java

```diff
@@ -111,6 +113,58 @@ public void contribute(OpenAPI openAPI, UriInfo uriInfo) {
 		}
 	}
 
+	private void _addRelationshipPaths(
```

Comment:
@ccorreagg these methods you've added look very similar to the code in `RelatedObjectEntryOpenAPIContributor`. Do you think we can extract common code to something like an `OpenAPIContributorUtil`?

Reply:
Yes, I will move it to the Util class